src/signframework/sign_cert/cca_functions.c

```diff
@@ -0,0 +1,2335 @@
+/* Copyright 2017 IBM Corp.
```

Comment:
The current design could use fixing but common routines like cca_function.c/h exist in the signframework directory and then the various tools just create a link to it .  So if there are changes in here they should be merged into the main copy and links inserted in sign_cert to ../cca_functions.c